recipe/meta.yaml

```diff
@@ -20,16 +20,17 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - patch     # [(not win) and (py==311)]
-    - m2-patch  # [win and (py==311)]
+    - patch     # [not win]
+    - m2-patch  # [win]
   host:
     - python
     - pip
-    - libprotobuf {{ version }}
+    - setuptools
+    - wheel
+    - libprotobuf  # pin handled through libprotobuf's run_exports
```

Comment:
Keep `- libprotobuf {{ version }}` in the host section and would move this line in the run section.